Sources/WeaverDI/ThreadSafeBuilderStoringDecorator.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ThreadSafeBuilderStoringDecorator.swift
+//  WeaverDIPackageDescription
+//
+//  Created by Bruno Mazzo on 5/31/18.
+//
+
+import Foundation
+
+class ThreadSafeBuilderStoringDecorator: BuilderStoring {
```

Comment:
Let's add a `final` here.

Reply:
Also, I'd prefer this to be named `ThreadSafeBuilderStoreDecorator`, the `ing` suffix being reserved for protocols.

Reply:
Done